docs-chef-io/content/inspec/resources/service.md

```diff
@@ -70,62 +77,93 @@ The following examples show how to use this Chef InSpec audit resource.
       it { should_not be_installed }
       it { should_not be_running }
     end
+```
 
-### Test Unix System V run levels
+### Test Unix SystemV run levels
 
 On targets that are using SystemV services, the existing run levels can also be checked:
```

Comment:
Which are using SystemV services - run levels?

the existing run levels can also be checked: - how?

What does - on targets mean?

Reply:
@ahasunos - pls check now

Reply:
Here **"on target"** means the target instance (could be any system) which is running on an older version of Unix. The newer Unix system supports systemd. It can be checked as shown in the example following the header.